volatility3/framework/symbols/windows/pdbconv.py

```diff
@@ -548,6 +560,11 @@ def omap_lookup(self, address):
 
     def name_strip(self, name):
         """Strips unnecessary components from the start of a symbol name."""
+        if name.startswith("?") and len(name.split("@@")) == 3:
```

Comment:
So what do you think we should do in case of a duplicate?

Reply:
If a duplicate is detected the non stripped name is used, and a warning is logged. If that warning is triggered in the future and reported by someone we can think about a different solution.

Reply:
Yeah, logging it seems like the best immediate possibility.  Might be worth doing it at warning so we get some visibility.  Since it will only happen at PDB generation we need to shout and scream (it would be worth testing how often we run into that situation, if it's not often we might want to abort so we don't produce broken ISF files)?

Reply:
Using the non-stripped name is likely to get us in trouble down the line if people start to rely on them in some instances.

Reply:
We've opted for the linkage name always being the unstripped one, because the main one will be the one that gets looked up, and we wanted to make that to be the main one people used (although, ideally not with parameters, etc).  So let's still try to use the stripped name, but as I say, in the case of duplicates we need to see if it's common or not.  If not, then bail out, if so then shout and scream and we need to think of how best to architect it (probably with input from @npetroni )...

Reply:
sure.